src/ejabberd_sm.erl

```diff
@@ -219,7 +219,7 @@ route(From, To, Acc, El) ->
 -spec open_session(SID, JID, Info) -> ReplacedPids when
       SID :: 'undefined' | sid(),
       JID :: jid:jid(),
-      Info :: 'undefined' | [any()],
+      Info :: info(),
```

Comment:
is `[]` a new equivalent of `undefined`?

Reply:
Looking at the code, `undefined` is never passed as a value of `Info` argument, as far as I know.